src/sidebar/store/modules/real-time-updates.js

```diff
@@ -97,8 +97,8 @@ function receiveRealTimeUpdates({
       // group and reload all annotations and discard pending updates
       // when switching groups.
       if (
-        ann.group === groupSelectors.focusedGroupId(getState().base) ||
-        !viewerSelectors.isSidebar(getState().base)
+        ann.group === groupSelectors.focusedGroupId(getState()) ||
```

Comment:
Curious as to why neither `base` nor a namespace is needed here anymore?

Reply:
because the `viewer` selectors were moved out of the `base` namespace and into its own.  Now its selectors will operate @ the root level and they will have to drill back down into the `viewer` namespace

As a workout to make the module backwards compatible, I first moved everything to "base", and automatically scoped each selector in each module to "getState().base" when its bound in the create-store util file. That made everything work w/o any changes to the modules. When we start to move modules out, they then get, their selectors get scoped to "getState()" instead.

This is a special case here because we're not calling the selector directly from the `store` instance which has these param bindings already set up. So we have to pass in the state manually and so we manually have to change the parameter.